Source/WebCore/style/StyleScopeRuleSets.cpp

```diff
@@ -234,7 +234,11 @@ void ScopeRuleSets::appendAuthorStyleSheets(const Vector<RefPtr<CSSStyleSheet>>&
 
     for (auto& cssSheet : styleSheets) {
         ASSERT(!cssSheet->disabled());
-        builder.addRulesFromSheet(cssSheet->contents(), cssSheet->mediaQueries());
+        const Node* ownerNode = cssSheet->ownerNode();
+        if (ownerNode)
+            ownerNode = ownerNode->parentNode();
+        const ContainerNode* container = dynamicDowncast<ContainerNode>(ownerNode);
```

Comment:
```suggestion
        auto* container = dynamicDowncast<ContainerNode>(ownerNode);
```

Reply:
`const auto*`

Reply:
Sure, you should probably use some type of smart pointer for nodes btw

Reply:
Ok so not `auto*` then 😅